apps/pdc-frontend/src/styles/openforms.css

```diff
@@ -0,0 +1,3 @@
+.utrecht-form-container--openforms .utrecht-form-label--openforms.utrecht-form-label--radio {
+  --utrecht-form-label-radio-font-weight: var(--utrecht-typography-weight-scale-normal-font-weight);
```

Comment:
Hmm dit voelt weird...

Wil je niet liever zoiets? De [originele implementatie](https://github.com/nl-design-system/utrecht/blob/main/components/form-label/css/_mixin.scss#L30C1-L31C1) gebruiken?

```scss
  font-weight: var(--utrecht-form-label-radio-font-weight, var(--utrecht-form-label-font-weight));
```

of:

```scss
.utrecht-form-container--openforms .utrecht-form-label--openforms.utrecht-form-label--radio {
  @include utrecht-form-label--radio;
  }
```

Reply:
ik heb het neer gezet als de originele implementatie